pkg/model/workflow.go

```diff
@@ -70,10 +71,10 @@ func (w *Workflow) OnEvent(event string) interface{} {
 func (w *Workflow) UnmarshalYAML(node *yaml.Node) error {
 	// Validate the schema before deserializing it into our model
 	if err := (&schema.Node{
-		Definition: "workflow-root-strict",
+		Definition: "workflow-root",
 		Schema:     schema.GetWorkflowSchema(),
 	}).UnmarshalYAML(node); err != nil {
-		return err
+		return errors.Join(err, fmt.Errorf("Actions YAML Schema Validation Error detected:\nFor more information, see https://nektosact.com/usage/schema.html."))
```

Comment:
```suggestion
		return errors.Join(err, fmt.Errorf("Actions YAML Schema Validation Error detected:\nFor more information, see: https://nektosact.com/usage/schema.html"))
```

Otherwise you get people who visit [https://nektosact.com/usage/schema.html.](https://nektosact.com/usage/schema.html.) and see:
> act - User Guide | Manual | Docs | Documentation
> 
> [Document not found (404)](https://nektosact.com/#document-not-found-404)
This URL is invalid, sorry. Please use the navigation bar or search to continue.